fiftyone/core/dataset.py

```diff
@@ -3193,8 +3193,9 @@ def delete_view(self, name):
         view_doc = self._get_view_doc(name, pop=True)
         self._doc.save()
 
-    def delete_views(self):
+    def delete_saved_views(self):
         """Deletes all saved views from this dataset."""
+        print("Deleting saved views: ", self.list_saved_views())
         self._doc.saved_views = []
         self._doc.save()
 
```

Comment:
You should rename private saved view-related methods here too, including:
- `_get_view_doc`
- `_views`
- `_reorder_views`
- `_validate_view_name`

Reply:
removed `_reorder_views` per the discussion earlier